src/core/eme/index.ts

```diff
@@ -156,19 +155,7 @@ function createEME(
    *
    * @type {Observable}
    */
-  const encrypted$ = onEncrypted$(video)
-    .distinctUntilChanged(function isSameEncryptedEvent(prevEvt, evt) {
-      if (
-        evt.initData == null || prevEvt.initData == null ||
-        evt.initDataType !== prevEvt.initDataType
-      ) {
-
-        return false;
-      }
-      const evtHash = hashInitData(new Uint8Array(evt.initData));
-      const prevHash = hashInitData(new Uint8Array(prevEvt.initData));
-      return evtHash === prevHash;
-    });
+  const encrypted$ = onEncrypted$(video);
```

Comment:
So do we manage the case where multiple encrypted event are emitted with the same init data and init data type?